src/components/AccountWarning/AccountWarning.tsx

```diff
@@ -54,6 +54,25 @@ export function AccountWarning() {
         }
     }, [user, user?.has_active_warning_flag]);
 
+    const mainGoban = (window as any)["global_goban"];
+
+    if (location.pathname.indexOf("game/") > 0) {
+        // On the Game page, we need to delay displaying the warning until the game is over
+        if (liveGameInProgress) {
+            return null;
+        } else if (mainGoban?.engine.phase === "play" && mainGoban.engine.game_id) {
```

Comment:
Should this be `phase !== "finished"`?  Do you want the message to show up during `"stone removal"`?

Reply:
I pondered that.    I felt being specific was safer, at the "expense" of showing in stone removal.   I don't know what other phases are possible (it's not typed, right?)  so allowing it to block on some phase like "paused" for example seems "not robust".    And ... stone removal is not time-pressured.  That was my thinking.

Reply:
> it's not typed, right?

https://github.com/online-go/goban/blob/9d83b03f00090f1bbf73b23577ede26225d5122c/src/GoEngine.ts#L43

> And ... stone removal is not time-pressured.

There is a timer on stone removal, but I agree it's not the same at in game.  Not including it seems reasonable, I just wanted to check!